plugin/src/main/scala/org/flywaydb/play/PlayInitializer.scala

```diff
@@ -113,6 +113,9 @@ class PlayInitializer @Inject() (implicit app: Application, webCommands: WebComm
     webCommands.addHandler(flywayWebCommand)
 
     for (dbName <- allDatabaseNames) {
+      if (flywayConfigurations(dbName).cleanOnStart) {
```

Comment:
@BrandonArp 
applyDownEvolutions is the configuration only for production and it exists to avoid unintentional destructive behavior in production.